radixdlt/src/integration/java/com/radixdlt/integration/distributed/deterministic/DeterministicAlwaysSyncedLedgerModule.java

```diff
@@ -44,7 +43,7 @@ Ledger syncedExecutor() {
 		return new Ledger() {
 			@Override
 			public PreparedCommand prepare(Vertex vertex) {
-				return PreparedCommand.create(0, Hash.ZERO_HASH);
+				return PreparedCommand.create(0, vertex.getQC().getTimestampedSignatures().weightedTimestamp());
```

Comment:
Given that this is a mocked Ledger would prefer to see this be 0L to emphasize/prove that consensus does not depend on the result of prepare().